client/style.css

```diff
@@ -391,6 +392,32 @@ h1{
   font-weight: 600;
 }
 
+.row-of-style-thumbnails-container {
+  position: relative;
+  width: 100%;
+  padding-top: 25%;
+}
+
+.row-of-style-thumbnails {
+  position: absolute;
+  top: 0;
+  display: flex;
+  flex-direction: row;
+  width: 100%;
+}
+
+.style-thumbnail-container {
+  min-width: 25%;
+  max-width: 25%;
+  position: relative;
+}
+
+.style-thumbnail {
```

Comment:
I wondered how you'd crop the thumbnails to be circular. Looking forward to seeing it!